<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageSession</class>
 <widget class="QWidget" name="PageSession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_top">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>409</width>
         <height>1600</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>1600</height>
           </size>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>311</width>
             <height>371</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="sessionPushButton">
                <property name="text">
                 <string>连接</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="update">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>更新配置</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QPushButton" name="modeTitle">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton:pressed{
color: rgb(255, 38, 0);
border-style: none
}
QPushButton{
border-style: none
}</string>
                </property>
                <property name="text">
                 <string notr="true">mode</string>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QComboBox" name="modeComboBox">
                <item>
                 <property name="text">
                  <string notr="true">peer</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">client</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="leftMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QPushButton" name="connectTitle">
                  <property name="font">
                   <font>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton:pressed{
color: rgb(255, 38, 0);
border-style: none
}
QPushButton{
border-style: none
}</string>
                  </property>
                  <property name="text">
                   <string notr="true">connect</string>
                  </property>
                  <property name="checkable">
                   <bool>false</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="connectAdd">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string notr="true">+</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="connectDel">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="connectListWidget">
                <item>
                 <property name="text">
                  <string notr="true">tcp/127.0.0.1:7447</string>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>10</number>
                </property>
                <property name="leftMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QPushButton" name="listenTitle">
                  <property name="font">
                   <font>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton:pressed{
color: rgb(255, 38, 0);
border-style: none
}
QPushButton{
border-style: none
}</string>
                  </property>
                  <property name="text">
                   <string notr="true">listen</string>
                  </property>
                  <property name="checkable">
                   <bool>false</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="listenAdd">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string notr="true">+</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="listenDel">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="listenListWidget">
                <item>
                 <property name="text">
                  <string notr="true">tcp/127.0.0.1:8000</string>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>16</pointsize>
                  </font>
                 </property>
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsEditable|ItemIsEnabled</set>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_level1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextBrowser" name="documentationTextBrowser"/>
      <widget class="QTextBrowser" name="jsonTextBrowser">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
